pkgs/development/python-modules/pyspinel/default.nix

```diff
@@ -1,15 +1,21 @@
-{ buildPythonPackage, fetchPypi, lib, future, pyserial, ipaddress }:
+{ buildPythonPackage, fetchFromGitHub, lib, future, pyserial, ipaddress }:
 
 buildPythonPackage rec {
   pname = "pyspinel";
   version = "1.0.1";
```

Comment:
if you're using a commit, then you should do `unstable-YYYY-MM-DD` as the version, where the date is the date of the commit.

Reply:
Thank you, review and pointers much appreciated.
Amended and force-pushed.